### PR TITLE
Scope text posted on behalf of @jhdalek55.

### PR DESCRIPTION
**Pull request by pattivacek**
I have not yet reviewed this myself; I just formatted it for the PR.

### CODE REVIEW

**Comment by jhdalek55** [comment]
Thanks, Patrick for getting this piece in place.

**Comment by patrickvacek** [comment]
I've now reviewed the text and put my corrections into a new commit in this same PR.

**Comment by jhdalek55** [comment]
Yes, I saw the comments. Thanks.

Lois

File: uptane-standard.md [hunk 1]
```diff
@@ -147,15 +147,42 @@ A standards document that can guide the safe design, integration and deployment
...
+This document sets guidelines for implementing Uptane in most systems capable of updating software on connected units in cars. In this section, we define the scope of that applicability by providing sample use cases and possible exceptions, aspects of update security that are not applicable to Uptane, and the design requirements governing the preparation of these standards.
 
 ### Use Cases
 
-TODO
+The following use cases provide a number of scenarios illustrating the manner in which software updates could be accomplished using Uptane.
+
+#### OEMs initializing Uptane at the factory using SOTA
+
+Bob, who works for an OEM, is overseeing the installation of Uptane on new vehicles at a manufacturing plant. He starts with preparing the ECUs by adding the following components: code to perform full and partial verification, the latest copy of the relevant metadata, the public keys, and the latest time, signed by the time server. His implementation would be considered Uptane-compliant if:
```
Comment (plapczyn) [inline uptane-standard.md]: Is it valuable to use individuals names for our usecases? Most of these things wouldn't be done by a sole individual and is many of the cases the "person" is actually a large corporation with many people involved. I'm questioning the utility of this, versus identifying having identified roles such as OEM, Tier1, etc.

Comment (jhdalek55) [reply uptane-standard.md]: I was going by the examples I had, which used names. I'm fine with switching it to just roles.

**Comment by iramcdonald** [comment]
Hi Phil,

The examples Lois had came from an IEEE Printer Working Group
protocol spec.  They were, of course, meant to refer to individuals
rather than simply roles.

But for Uptane, only actor/role names (i.e., Release Engineer) are
appropriate.

Cheers,
- Ira
Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
Jan-April: 579 Park Place  Saline, MI  48176  734-944-0094
May-Dec: PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by jhdalek55** [comment]
Thanks for the clarification, Ira. Phil, if you want to go ahead and
eliminate the names, be my guest. Otherwise, i will take care of it next
week.

Lois

**Comment by tkfu** [comment]
I've rendered this and uploaded it in [HTML](https://uptane.github.io/uptane-standard/PR-13/uptane-standard.html) and [plain text](https://uptane.github.io/uptane-standard/PR-13/uptane-standard.txt), for easier reviewing. (cc @iramcdonald)

File: uptane-standard.md (hunk 1 above)
Comment (tkfu) [inline uptane-standard.md]: Might be good to specify which public keys we're talking about.

File: uptane-standard.md [hunk 2]
```diff
@@ -147,15 +147,42 @@ A standards document that can guide the safe design, integration and deployment
...
+
+#### Dealership updating individual ECUs on demand
+
+Dana runs a dealership for a major OEM. The OEM has issued a recall to address a problem with a keyless entry device that has been locking people out of their cars. Individual owners are bringing in a revised image on a flash drive that was sent to them from the manufacturer via courier mail. To carry out this update, the OEM would first have to delegate to Dana the authority to sign the metadata that would need to accompany the image on the flashdrive. He would then follow the same procedures used by Clark in the example above.
```
Comment (tkfu) [inline uptane-standard.md]: I don't understand this use case at all. Why would a manufacturer send out updates on a USB stick, then tell customers to bring the USB stick to a dealership and have the dealership sign the metadata and add it to the image repository?

Comment (jhdalek55) [reply uptane-standard.md]: The use cases are being addressed by Andre, who agreed to revise this portion of the text. These comments can be addressed in the new text.

**Pull request merged**